README.md

```diff
@@ -386,6 +400,7 @@ The power of EB is that it will automatically update our environment to satisfy
 1. Tell EB to update your environment accordingly:
 
   ```
+  $ git add --all
   $ eb deploy --staged
```

Comment:
Interesting, this seems like not a best practice be cause you cant track the changes to a commit hash